Microsoft.AVS.VMFS/Microsoft.AVS.VMFS.psm1

```diff
@@ -462,3 +462,38 @@ function Restore-VmfsVolume {
 
     $Esxi | Get-VMHostStorage -RescanVMFS -ErrorAction stop | Out-Null
 }
+
+<#
+    .SYNOPSIS
+     This function removes the static iSCSI configurations from a specified Esxi Host
+
+    .PARAMETER VMHostName
+     Name of the VMHost (ESXi server)
+
+    .EXAMPLE
+     Remove-VMHostStaticiSCSITargets -VMHostName "vmhost1"
+
+    .INPUTS
+     VMHostName
+
+    .OUTPUTS
+     None
+#>
+function Remove-VMHostStaticiSCSITargets {
+    [CmdletBinding()]
+    [AVSAttribute(10, UpdatesSDDC = $false)]
+    Param (
+        [Parameter(
+                Mandatory=$true,
+                HelpMessage = 'VMHost name')]
+        [ValidateNotNull()]
+        [String]
+        $VMHostName
+    )
+
+    $VMHost = Get-VMHost $VMHostName -ErrorAction Ignore
+    if (-not $VMHost) {
+        throw "VMHost $VMHostName does not exist."
+    }
+    $VMHost| Get-VMHostHba -Type iScsi | Get-IScsiHbaTarget | Where {$_.Type -eq "Static"} | Remove-IScsiHbaTarget -Confirm:$false
```

Comment:
I'm wondering if this needs to be more specific that just `Static`? Could we have multiple vendors of static iSCSI targets?

Reply:
Good point. Updated the code to filter the removed static addresses